src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -661,7 +661,8 @@
             "PreferredExtendedPanID",
             "ThreadNetworks",
             "ThreadNetworkTableSize"
-        ]
+        ],
+        "Service Area": ["CurrentLocation", "EstimatedEndTime"]
```

Comment:
And FeatureMap?

Reply:
Yes. We're managing the FeatureMap. Added [here](https://github.com/project-chip/connectedhomeip/pull/33991/commits/9dbe4a4ecaa87abf7731c6cee17676e2a6af18c7).